kubernetes/cray-tapms-crd/files/tapms.hpe.com_tenants.yaml

```diff
@@ -158,6 +158,25 @@ spec:
                 type: array
               state:
                 type: string
+              tenantkms:
```

Comment:
A note for future consideration. We might want to move the KMS section under tenantresources and allow multiple KMSs.

Reply:
I will add a jira item to address this.